controller/argument_value_resolver.rst

```diff
@@ -40,6 +40,10 @@ Symfony ships with four value resolvers in the HttpKernel component:
 
     Prior to Symfony 3.1, this logic was resolved within the ``ControllerResolver``.
     The old functionality is rewritten to the aforementioned value resolvers.
+    
+.. note::
+    If your argument is a Doctrine Entity you will need to create a 
+    :doc:`param converter </bundles/SensioFrameworkExtraBundle/annotations/converters>`
```

Comment:
I think the note should explicit that param converters are executed before argument value resolvers.

This can be useful though since the entity is accessible through request attributes for any custom resolvers.